sarracen/kernels.py

```diff
@@ -0,0 +1,41 @@
+from math import pi
```

Comment:
Better to
`import numpy as np`
and then use np.pi in the code below. Numpy is generally preferred over the default Python math library as it is usually faster and more accurate.

Reply:
I didn't know that! Does the same also apply for trigonometric functions & other kinds of mathematical operations?

Reply:
Yes, and they work extremely well with pandas. Numpy functions all allow vectorization, so you could use np.sin() of a pandas series, for example.